pkg/controllers/disruption/controller.go

```diff
@@ -251,6 +263,21 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command,
 	return nil
 }
 
+func evictionReason(m Method, nodeClaim *v1beta1.NodeClaim) string {
```

Comment:
Is there a reason we prefer passing in a method vs just including this in the methods' pointer receivers